config/test_glyphs-glyf_colr_1.py

```diff
@@ -585,74 +698,37 @@ def _make_test_glyph(self, param_set, position, accessor):
         )
 
 
-def _upem_box_pen():
-    pen = TTGlyphPen(None)
-    pen.moveTo((0, 0))
-    pen.lineTo((0, _UPEM))
-    pen.lineTo((_UPEM, _UPEM))
-    pen.lineTo((_UPEM, 0))
-    pen.closePath()
-
-    return pen
+def _get_shape_pen(shape_name: str, transform: tuple = None) -> list[tuple]:
+    if shape_name not in SHAPES:
+        return []
```

Comment:
it doesn't look like it's returning a `list[tuple]` as the return type annotations says, below it's returning a TTGlyphPen.
Maybe make the absence of shape_name not in SHAPES an error (it's most likely a typo) and change the annotation to `-> TTGlyphPen`

Reply:
Made than an error and fixed the return type annotation.